source/tutorials/cloud-storage/private-swift-container/index.rst

```diff
@@ -10,6 +10,14 @@ In this guide, we'll link a private `OpenStack Swift <https://wiki.openstack.org
    :backlinks: none
    :local:
 
+.. container:: alert alert-warning
```

Comment:
Same suggestion here as above (... hey, should we make this an includable fragment to avoid copy-paste?)